src/ImageSharp/Formats/PixelTypeInfo.cs

```diff
@@ -16,7 +16,7 @@ public class PixelTypeInfo
         /// </summary>
         /// <param name="bitsPerPixel">Color depth, in number of bits per pixel.</param>
         /// <param name="alpha">Tthe pixel alpha transparency behavior.</param>
-        internal PixelTypeInfo(int bitsPerPixel, PixelAlphaRepresentation? alpha = null)
+        public PixelTypeInfo(int bitsPerPixel, PixelAlphaRepresentation? alpha = null)
```

Comment:
@antonfirsov I'm wondering whether we should avoid the nullable parameter here and either provide an overload or break things for V2.

I'm not overly concerned about the implications of making this public because I'd like to do a thorough review for V3 to make sure we capture everything we should.

Reply:
I think this should be refactored into 2 constructors for now. In long term, I would explore [init-only setters](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/init) for such API-s.

Reply:
As for the V3 API concern: I think minor breaking changes are fine in advanced API-s.

Reply:
Yeah, that keeps with our pattern for avoiding nullable parameters when possible. @jz5 Could you please update the `PixelTypeInfo` constructor to use separate overloads with and without the `PixelAlphaRepresentation` (not nullable) parameter. Thanks!

Reply:
I've updated it. How about this?